picky-server/Cargo.lock

```diff
@@ -0,0 +1,4082 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I added this file to git because picky-server is a binary and we weren't keeping track of the lock file, but maybe this file should be reverted instead? @CBenoit what do you think?